### PR TITLE
add: struct and handling of klines from binance

### PR DESCRIPTION
**Pull request by kitplummer**
Basic handling of https://binance-docs.github.io/apidocs/spot/en/#kline-candlestick-data .  Tried to follow the basic struct pattern and added a test as well.

### CODE REVIEW

File: lib/binance.ex [hunk 1]
```diff
@@ -97,6 +97,33 @@ defmodule Binance do
...
   end
 
+  @doc """
+  Retrieves klines for a symbol, provided a given interval
+  """
+
+  def get_klines(symbol, interval, limit \\ 500) when is_binary(symbol) do
+    case HTTPClient.get_binance("/api/v3/klines?symbol=#{symbol}&interval=#{interval}&limit=#{limit}") do
+      {:ok, data} ->
+        #IO.inspect data
```
Comment (dvcrn) [inline lib/binance.ex:107]: Can you remove the commented out code?

**Comment by dvcrn** [comment]
Thanks for the pull request! I added some comments

File: lib/binance/kline.ex [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    :volume,
+    :close_time,
+    :quote_asset_volume,
+    :number_of_trades,
+    :taker_buy_base_asset_volume,
+    :taker_buy_quote_asset_volume,
+    :ignore
+  ]
+
+  def new(list) do
```
[the remark this answers is not in the document]
Comment (kitplummer) [reply lib/binance/kline.ex:38]: Hrm.  I tried to do it, but Binance actual returns the kline info as a List, not a Map.  Take a peek here: https://binance-docs.github.io/apidocs/spot/en/#kline-candlestick-data - maybe ExConstructor _is_ smart enough to figure out the struct load, but I'm not sure how to do it.

[the remark this answers is not in the document]
Comment (dvcrn) [reply lib/binance/kline.ex:38]: Oh you're right. Don't know why I expected a struct

**Comment by dvcrn** [comment]
LGTM! Thanks for the contribution

**Comment by dvcrn** [comment]
Released in https://github.com/dvcrn/binance.ex/tree/v0.8.0

**Pull request merged**